src/Controller/Api/FileManager.php

```diff
@@ -323,70 +326,35 @@ public function downloadFile(File $file, Datastore $datastore): Response
      *
      * @return Response
      */
-    public function downloadZipAllFiles(DatasetSubmission $datasetSubmission): Response
+    public function downloadZipAllFiles(DatasetSubmission $datasetSubmission, LogActionItemEventDispatcher $logActionItemEventDispatcher, Request $request): Response
```

Comment:
The docblock can be fixed up some. Order, remove unneeded (or add) params. @Route should be last.